src/app/store/ha-info/ha-info.effects.ts

```diff
@@ -42,7 +43,10 @@ export class HaInfoEffects {
   loadUpgradePendingState = createEffect(() => this.actions$.pipe(
     ofType(haStatusLoaded),
     mergeMap(({ haStatus }) => {
-      if (haStatus.hasHa && haStatus.reasons.length === 0) {
+      if (
+        (haStatus.hasHa && haStatus.reasons.length === 0)
+        || (haStatus.reasons?.includes(FailoverDisabledReason.MismatchVersions))
```

Comment:
The check is not correct. We only want to allow this proceed when MismatchVersion is the only disabled reason, not one of them.

Reply:
gonna fix